packages/www/docs/guides/start-live-streaming/api-key.mdx

```diff
@@ -11,18 +11,42 @@ You need an API key to make requests of the Livepeer.com API. You do not need to
 create an API key if you are creating and configuring live streams using only
 the Livepeer.com Dashboard.
 
+When you create a new API key, copy and secure the value displayed to you. This
+is the sole secret you will have to use when making requests to the Livepeer.com
+API, so share and store it with caution.
+
 Login to the Livepeer.com Dashboard, and navigate to
 [the API keys page within the Developers tab](https://livepeer.com/dashboard/developers/api-keys).
 On this page, you can manage your API keys including creating or deleting them
 and monitoring their usage.
 
-When you create a new API key, copy and secure the value displayed to you. This
-is the sole secret you will have to use when making requests to the Livepeer.com
-API, so share and store it with caution. We currently allow you to reveal the
-secret of any existing API key but this behavior may change in the future, so
+## How to get an API Key using the Livepeer.com Dashboard
+
+Make sure you register Livepeer Video Service Account and get API Key
+
+- Create a free [Livepeer Video Service Account](https://livepeer.com/register)
+
+![Api Guide 1](/img/docs/api-guide1.png)
+
+- Click `Create API Key`
+
+![Api Guide 1](/img/docs/api-guide2.png)
+
+- A popup should show up. Enter the name for your API key and then Click `Create`.
+
+![Api Guide 3](/img/docs/api-guide3.png)
```

Comment:
This screenshot is outdated since it doesn't have the CORS checkbox. This how-to is also not mentioning anything about CORS, which I think it must. Would you mind adding some information about it here? Maybe a screenshot about it as well, can copy the ones from the CORS section below (and remove from it, if it makes sense).

Reply:
@codentell can you update the screenshot and resubmit the PR?

Reply:
@pglowacky i can